package.json

```diff
@@ -84,7 +84,7 @@
                 "icon": "media/dog.svg"
             },
             {
-                "command": "vscode-pets.throw-with-mouse",
+                "command": "vscode-pets.throwBallWithMouse",
```

Comment:
Why did you change the command ID to camelCase?

Reply:
Sorry it had the same name as a variable in extension.ts so I presumed that it had to be named the same.